fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowMaterializedViewsStmt.java

```diff
@@ -34,19 +34,27 @@
 public class ShowMaterializedViewsStmt extends ShowStmt {
     private static final ShowResultSetMetaData META_DATA =
             ShowResultSetMetaData.builder()
-                    .addColumn(new Column("id", ScalarType.createVarchar(50)))
-                    .addColumn(new Column("name", ScalarType.createVarchar(50)))
-                    .addColumn(new Column("database_name", ScalarType.createVarchar(20)))
-                    .addColumn(new Column("refresh_type", ScalarType.createVarchar(10)))
-                    .addColumn(new Column("is_active", ScalarType.createVarchar(10)))
-                    .addColumn(new Column("last_refresh_start_time", ScalarType.createVarchar(20)))
-                    .addColumn(new Column("last_refresh_finished_time", ScalarType.createVarchar(20)))
-                    .addColumn(new Column("last_refresh_duration", ScalarType.createVarchar(20)))
-                    .addColumn(new Column("last_refresh_state", ScalarType.createVarchar(20)))
-                    .addColumn(new Column("inactive_code", ScalarType.createVarchar(20)))
-                    .addColumn(new Column("inactive_reason", ScalarType.createVarchar(1024)))
-                    .addColumn(new Column("text", ScalarType.createVarchar(1024)))
-                    .addColumn(new Column("rows", ScalarType.createVarchar(50)))
+                    .column("id", ScalarType.createVarchar(50))
+                    .column("database_name", ScalarType.createVarchar(20))
+                    .column("name", ScalarType.createVarchar(50))
+                    .column("refresh_type", ScalarType.createVarchar(10))
+                    .column("is_active", ScalarType.createVarchar(10))
+                    .column("partition_type", ScalarType.createVarchar(16))
+                    .column("task_id", ScalarType.createVarchar(20))
+                    .column("task_name", ScalarType.createVarchar(50))
+                    .column("last_refresh_start_time", ScalarType.createVarchar(20))
+                    .column("last_refresh_finished_time", ScalarType.createVarchar(20))
+                    .column("last_refresh_duration", ScalarType.createVarchar(20))
+                    .column("last_refresh_state", ScalarType.createVarchar(20))
+                    .column("force_refresh", ScalarType.createVarchar(8))
+                    .column("start_partition", ScalarType.createVarchar(1024))
+                    .column("end_partition", ScalarType.createVarchar(1024))
+                    .column("base_refresh_partitions", ScalarType.createVarchar(1024))
+                    .column("mv_refresh_partitions", ScalarType.createVarchar(1024))
```

Comment:
better use last_xxx to represent the current state of the refresh materialized view.